### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@msehnout PTAL :)

**Comment by msehnout** [comment]
Renewal of the RSS feeds souds like a good idea :). LGTM

File: release_monitor/release_monitor.py [hunk 1]
```diff
@@ -114,6 +114,8 @@ def run(self):
         self.log.info("Registered signal handler for liveness probe")
 
         while True:
+            self.renew_rss_feeds()
```
Comment (tisnik) [inline release_monitor/release_monitor.py]: do I understand correctly that if the rss_feed is empty at beginning, the notification is send for all packages?

File: openshift/template.yaml [hunk 2]
```diff
@@ -143,7 +143,7 @@ parameters:
 - description: Variable which defines time between requests for updates from registries
   displayName: Sleep interval
   name: SLEEP_INTERVAL
-  value: "60"
+  value: "10"
```
Comment (tisnik) [inline openshift/template.yaml]: IIRC 60 minutes was set because people were not happy with tons of logs on production (this was another issue different from logging everything everytime ;)

Comment (msehnout) [reply openshift/template.yaml]: Exactly, this should not have any impact, if we are logging only new versions. (It does not matter if we get 5 logs every 10 minutes or 30 logs once an hour)...

File: release_monitor/release_monitor.py (hunk 1 above)
Comment (msehnout) [reply release_monitor/release_monitor.py]: Based on my testing. The implementation is broken at type system level. For example: the old RSS feed is not an empty list, but None, so the comparison fails in the first cycle; if you use the first feed content as an initial value for the old RSS feed, the issue is replaced by a different one. More specifically the "entry" type does not implement comparison, so you cannot sort it. If you remove the sorting, it works again, but there is a chance that the feed itself is not sorted, so the comparison might give undefined result.

Long story short, it is hard to reason about this code :-).

**Comment by msrb** [comment]
Thanks for the feedback @tisnik and @msehnout. :+1:
